TodoApp/TodoApp.Api/App_Start/RoutesConfig.cs

```diff
@@ -2,9 +2,9 @@
 
 namespace TodoApp.Api
 {
-    internal static class RoutesConfig
+    public static class RoutesConfig
     {
-        internal static readonly string ApiV2Route = "ApiV2";
+        public static readonly string ApiV2Route = "ApiV2";
```

Comment:
While the class is supposed to be internal, this is actually potentially correct - I will leave it up to you. There are 2 levels of access modifiers (class-wise and member-wise) that are a bit different. 

Class-wise modifiers are simple: internal classes are used only within the assembly, while public classes form the public API of an assembly (and for clarity and maintainability, it should be as small as possible)

Member-wise modifiers are tougher: they can either be approached from the same standpoint as the class-wise, however, one quite often repeats themselves and they seem to lose their purpose; or they can be approached as a finer tuning mechanism that is not affected by the class-wise modifier - it the other way, imagine the class was the whole system of yours, what of its member would be `public`, `internal`, `protected`, `private`?